src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -725,6 +725,13 @@ def load_arguments(self, _):
             c.argument('identity_role', options_list=['--role'], arg_group=arg_group, help="Role name or id the system assigned identity will have")
             c.ignore('identity_role_id')
 
+    with self.argument_context('vm auto-shutdown') as c:
+        c.argument('disable', arg_type=get_three_state_flag(), help='Disable auto-shutdown for VM')
+        c.argument('email', help='The email recipient to send notifications to (can be a list of semi-colon separated email addresses)')
+        c.argument('time', help='The time of day the schedule will occur. Format: hhmm. Example: 1730')
```

Comment:
only can do auto-shutdown by day ? Can we do it by week or month ?

Reply:
The service can. But I have not added to CLI. I copy the portal design. Portal covers a frequently used subset of parameters. It only support time of day. This is the most common usage.